packages/wpcom-checkout/src/payment-methods/existing-credit-card.tsx

```diff
@@ -114,6 +120,10 @@ function ExistingCardLabel( {
 			</div>
 			<div className="existing-credit-card__logo payment-logos">
 				<PaymentLogo brand={ brand } isSummary={ true } />
+				<br />
+				<span>Postal Code: { 'postalCode' }</span>
+				<br />
+				<span>Country Code: { 'countryCode' }</span>
```

Comment:
I know this is still in progress, but just wanted to mention that I expected the `PaymentMethodEdit` component to be here too (because the idea is that you need to be able to edit the tax location fields while assigning an existing credit card to a subscription, and we'll block assigning if the fields aren't filled out).

That said, I like that you are adding it to the overall payment method management page also; it does seem like that could be useful.

Reply:
This is being added in.